### PR TITLE
[9.0][MIG] migrate module portal_partner_select_all

### PR DESCRIPTION
**Pull request by robinkeunen**

### CODE REVIEW

**Review by rafaelbn — APPROVED** [review]

File: portal_partner_select_all/wizard/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from . import portal_wizard
```
Comment (NL66278) [inline portal_partner_select_all/wizard/__init__.py:1]: Still need a coding line in 9.0 (check all the init files. Would also prefer the license lines here).

File: portal_partner_select_all/wizard/portal_wizard.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (NL66278) [inline portal_partner_select_all/wizard/portal_wizard.py:3]: Make license lines https:

**Review by NL66278 — CHANGES_REQUESTED** [review]
:+1: Mostly good, few changes would make it perfect

File: portal_partner_select_all/__openerp__.py [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
+# Copyright 2019 Coop IT Easy SCRL fs - Robin keunen
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     'name': 'Portal Partner Select All',
-    'version': '10.0.1.0.1',
+    'version': '9.0.1.0.1',
     'category': 'Custom',
     'author': 'Tecnativa,'
+              'Coop IT Easy,'
```
Comment (chienandalu) [inline portal_partner_select_all/__openerp__.py:11]: Sorry, but this kind of simple migrations, as welcome as they ar, can't entitle your work as author.

File: portal_partner_select_all/__openerp__.py [hunk 4]
```diff
@@ -1,12 +1,14 @@
 # -*- coding: utf-8 -*-
 # Copyright 2018 Tecnativa S.L. - David Vidal
+# Copyright 2019 Coop IT Easy SCRL fs - Robin keunen
```
Comment (chienandalu) [inline portal_partner_select_all/__openerp__.py:3]: The same here. No relevant changes were made to add your copyright

**Review by chienandalu — CHANGES_REQUESTED** [review]
Looks fine overall. Please attend comments regarding the shebang remove and my remarks about authorship. Squash your commits when the final change are done.

**Comment by robinkeunen** [comment]
cf #812, the repository this PR used was deleted.

**Pull request closed without merging**